<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAboutSmashtein</class>
 <widget class="QDialog" name="DialogAboutSmashtein">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>221</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>481</width>
    <height>221</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>221</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Smashtein Garbage</string>
  </property>
  <widget class="QGraphicsView" name="imgAboutSmashtein">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>150</width>
     <height>179</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>179</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>179</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Smashtein Garbage by : 

Uga, The-Doc, HighRonn, Joora, Sheep7, Pingouin




IMAC - Promotion  2011</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
